docs/recipes/delete-files-folder.md

```diff
@@ -70,13 +70,20 @@ var del = require('del');
 var vinylPaths = require('vinyl-paths');
 
 gulp.task('clean:tmp', function () {
+  var vp = vinylPaths();
   return gulp.src('tmp/*')
+    .pipe(vp)
```

Comment:
I have test the all usecase , it is right. See [this](https://cnodejs.org/topic/5552fc48c62bb7483db55101).